msg/js/en.js

```diff
@@ -152,6 +152,8 @@ Blockly.Msg["OPERATORS_LT"] = "%1 < %2";
 Blockly.Msg["OPERATORS_EQUALS"] = "%1 = %2";
 Blockly.Msg["OPERATORS_AND"] = "%1 and %2";
 Blockly.Msg["OPERATORS_OR"] = "%1 or %2";
+Blockly.Msg["OPERATORS_RELABEL_AND"] = "And";
```

Comment:
these should probably be lower case to be consistent with the blocks

Reply:
I was debating over which was right here...thanks for the input! Resolved here: 67d6b127bd00b5150a43e80c658547f23ccb0144 Also thanks for the details on making translation changes.